views/js/qtiCreator/editor/mathInput/mathInput.js

```diff
@@ -55,30 +55,98 @@ define([
      * src: https://github.com/mathquill/mathquill/issues/74
      */
     var allTools = {
-            frac:       { label: 'x/y',         latex: '\\frac',        fn: 'cmd',      desc: __('Fraction') },
+            // Basic operations and functions
             sqrt:       { label: '&radic;',     latex: '\\sqrt',        fn: 'cmd',      desc: __('Square root') },
+            frac:       { label: 'x/y',         latex: '\\frac',        fn: 'cmd',      desc: __('Fraction') },
             exp:        { label: 'x&#8319;',    latex: '^',             fn: 'cmd',      desc: __('Exponent') },
             log:        { label: 'log',         latex: '\\log',         fn: 'cmd',      desc: __('Log') },
             ln:         { label: 'ln',          latex: '\\ln',          fn: 'cmd',      desc: __('Ln') },
+
+            // Constants and brackets
             e:          { label: '&#8494;',     latex: '\\mathrm{e}',   fn: 'write',    desc: __('Euler\'s constant') },
             infinity:   { label: '&#8734;',     latex: '\\infty',       fn: 'cmd',      desc: __('Infinity') },
+            colon:      { label: ':',           latex: ':',             fn: 'write',    desc: __('Colon') },
             lbrack:     { label: '[',           latex: '\\lbrack',      fn: 'cmd',      desc: __('Left bracket') },
             rbrack:     { label: ']',           latex: '\\rbrack',      fn: 'cmd',      desc: __('Right bracket') },
```

Comment:
When I click on `[` or `]`, there's text not symbol in the editor:
![Screenshot_43](https://github.com/user-attachments/assets/abf456ff-d202-4d8d-a19f-20b9d64a3209)

Is it fine for you? If yes, ok. let's ignore this.